sdk/core/core/src/az_version.c

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "az_version.h"
+
+int az_core_version_get_major() { return AZ_VERSION_MAJOR; }
+int az_core_version_get_minor() { return AZ_VERSION_MINOR; }
+int az_core_version_get_patch() { return AZ_VERSION_PATCH; }
+const char* az_core_version_get_prerelease() { return AZ_VERSION_PRERELEASE; }
```

Comment:
Unit tests?

Reply:
Functions are removed

Reply:
I think it is probably still useful to add some sort of sanity test, for instance, something like this:

if (AZ_SDK_VERSION_PRERELEASE)
{
assert_equal(AZ_SDK_VERSION_STRING, AZ_SDK_VERSION_MAJOR.AZ_SDK_VERSION_MINOR.AZ_SDK_VERSION_PATCH-AZ_SDK_VERSION_PRERELEASE);
}
else
{
assert_equal(AZ_SDK_VERSION_STRING, AZ_SDK_VERSION_MAJOR.AZ_SDK_VERSION_MINOR.AZ_SDK_VERSION_PATCH);
}